.travis.yml

```diff
@@ -1,96 +1,60 @@
 language: cpp
+
+dist: trusty
```

Comment:
If there's some change in the current behaviour of travis, we still can drop the line.